kernel/os/src/os.c

```diff
@@ -111,7 +112,7 @@ os_idle_task(void *arg)
         ++g_os_idle_ctr;
 
         now = os_time_get();
-        if (OS_TIME_TICK_GT(now, sanity_last + sanity_itvl_ticks)) {
+        if (OS_TIME_TICK_GEQ(now, sanity_last + sanity_itvl_ticks)) {
```

Comment:
this changes sanity check frequency by one tick.
I think it could be left as it was except that line 139 could be change to
sanity_to_next = 0;

Reply:
now interval is accurate, disregard my previous comment